fiddler/datadog_checks/fiddler/check.py

```diff
@@ -0,0 +1,152 @@
+# import datetime
+# import json
+# from requests.exceptions import Timeout
+
+# import yaml
+import time
+
+import fiddler as fdl
+
+from datadog_checks.base import AgentCheck
+
+# from typing import Any
+
+
+# We will get one hours worth data in chunks and process them.
+bin_size = 3600
+
+# metrics_list = ['accuracy', 'histogram_drift', 'output_average', 'feature_average', 'traffic_count']
+metrics_list = ['accuracy']
+tags = list()
+
+# Connection parameters for Fiddler. Currently hard coded to get some testing done. Will be parameterizing this later.
+URL = 'https://demo.trial.fiddler.ai'
+ORG_ID = 'demo'
+AUTH_TOKEN = 'K4ph7ORDcIO2xVIEA6KxL1o1zHjZockgurhCOZOUSVs'
+
+client = fdl.FiddlerApi(url=URL, org_id=ORG_ID, auth_token=AUTH_TOKEN)
+
+
+def create_tags(**tags_in):
+    tags.clear()
+    tags.append("project:" + tags_in['project'])
+    tags.append("model:" + tags_in['model'])
+    if len(tags_in) > 2:
+        tags.append("feature:" + tags_in['feature'])
+
+
+class FiddlerCheck(AgentCheck):
+
+    # This will be the prefix of every metric and service check the integration sends
+    __NAMESPACE__ = 'fiddler'
+
+    def __init__(self, name, init_config, instances):
+        super(FiddlerCheck, self).__init__(name, init_config, instances)
+
+    #        self.base_url = self.instance.get('url')
+    #        self.api_key = self.instance.get('fiddler_api_key')
+    #        self.org = self.instance.get('organization')
+
+    #        print("Connecting to : ", self.base_url)
+    #        print("with org id : ", self.org)
+    #        print("and auth key : ", self.api_key)
+
+    #        client = fdl.FiddlerApi(url=self.base_url, org_id=self.org, auth_token=self.api_key)
+
+    # Use self.instance to read the check configuration
+
+    def check(self, _):
+        # Iterate through the projects and the models and push data into Fiddler
+        project_path = ['list_projects', ORG_ID]
+        result_all = client._call(project_path)
+        print("Projects: ", result_all["projects"])
```

Comment:
Instead of using `print`, you could use the default logger `self.log`

Reply:
Done. I was keeping the print statements for my local testing before I integrate with DD and get the logs inside the DD agent log files.